app.rb

```diff
@@ -228,12 +229,14 @@ def idp_public_key
     end
 
     def token(code)
+      logger.debug("DEBUG: in #token, token_endpoint = #{openid_configuration[:token_endpoint]}")
```

Comment:
Were we intending to add new logging in this pull request?

Reply:
That is an oversight; removing.  Thank you and again, apologies for not updating faster... my workflow should improve.

Reply:
> apologies for not updating faster... my workflow should improve.

No need to apologize! It certainly isn't my intent to rush any changes here.